CMake_VS2015.bat

```diff
@@ -1,9 +1,35 @@
 @ECHO OFF
 @echo:
 @echo:
-ECHO Generating Visual Studio Solution (64 bit)
+SET ARG=%1
+@echo:
 @echo:
+ECHO Generating Visual Studio Solution (64 bit)
 @echo:
+SET BUILDMODE="d3d9"
+IF DEFINED ARG (
+    IF "%ARG%"=="--opengl" (
+    	SET BUILDMODE="opengl"
+		ECHO OpenGL configuration selected.
+	)
+	IF "%ARG%"=="/opengl" (
+    	SET BUILDMODE="opengl"
+		ECHO OpenGL configuration selected.
+	)
+    IF "%ARG%"=="--d3d11" (
+    	SET BUILDMODE="d3d11"
+		ECHO Direct3D 11 configuration selected.
+	)
+    IF "%ARG%"=="/d3d11" (
+    	SET BUILDMODE="d3d11"
+		ECHO Direct3D 11 configuration selected.
+	)
+)
+IF %BUILDMODE%=="d3d9" (
+  ECHO Direct3D 9 configuration selected by default (Use /opengl or /d3d11 switches to specify OpenGL or DirectX 11)
+
```

Comment:
I like that you support switched on Windows :)  Ok, this bat file is about as complicated as it should ever be now, we need to move this to use the build scripts.

So, we'll have AtomicBuild.sh and AtomicBuild.bat, or something, which will be capable of generating binary AtomicEditor + deployment *and* VS/XCode/QtCreator workspaces, instead of having Build_AtomicEditor.bat and CMake_VS2015.bat for example